src/presence_analyzer/utils.py

```diff
@@ -52,17 +54,17 @@ def get_data():
     """
     data = {}
     with open(app.config['DATA_CSV'], 'r') as csvfile:
-        presence_reader = csv.reader(csvfile, delimiter=',')
+        presence_reader = csv.reader(csvfile, delimiter=str(','))
```

Comment:
I keep getting this error otherwise: 
![image](https://cloud.githubusercontent.com/assets/10250139/24951001/305ccbe0-1f73-11e7-87e9-ba0ccfe8b816.png)